server/src/modules/Question/question.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@nestjs/common';
+import { QuestionDto } from './dto/question.dto';
+import { Questions } from '../../entity/Questions';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+@Injectable()
+export class QuestionService {
+
+    constructor(
+        @InjectRepository(Questions)
+        private readonly questionRepository: Repository<Questions>,
+    ) { }
+
+    async findAll(): Promise<Questions[]> {
+        try {
+            return await this.questionRepository.find({});
```

Comment:
Is it need to pass an empty object to the find method?

Reply:
for finding all records no need, removed